src/admin-views/aggregator/origins/limit.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /** @var \Tribe__Events__Aggregator__Settings $settings */
 $settings             = tribe( 'events-aggregator.settings' );
-$global_limit_type    = tribe_get_option( 'tribe_aggregator_default_import_limit_type', 'range' );
+$global_limit_type    = tribe_get_option( 'tribe_aggregator_default_import_limit_type', 'count' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces

<sub>hash: 2565573280bb0a8275c414bb0e9571f4</sub>